weed/command/filer_remote_sync_dir.go

```diff
@@ -37,6 +37,14 @@ func followUpdatesAndUploadToRemote(option *RemoteSyncOptions, filerSource *sour
 
 	var lastLogTsNs = time.Now().UnixNano()
 	processEventFnWithOffset := pb.AddOffsetFunc(func(resp *filer_pb.SubscribeMetadataResponse) error {
+		if option.storageClass != nil {
+			storageClass := *option.storageClass
+			if storageClass == "None" {
```

Comment:
How about just use empty string to delete, but default to current default value?